ksp_plugin/plugin.cpp

```diff
@@ -482,12 +480,8 @@ void Plugin::IncrementPartIntrinsicForce(PartId const part_id,
 }
 
 void Plugin::PrepareToReportCollisions() {
-  for (auto const& pair : vessels_) {
-    Vessel& vessel = *pair.second;
-    // TODO(egg): we're taking the address of a parameter passed by reference
-    // here; but then I don't think I want to pass this by pointer, it's quite
-    // convenient everywhere else...
```

Comment:
I would keep the comment.

Reply:
Changed it to a note and documented the lifetime requirements in various places.